server/testUsers.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO public.users (
+	 name,
```

Comment:
hi
we need to have ids populated for test data
otherwise, it won't be consistent/linked